azure-functions/acbs-function/mappings/facility/code-value-transaction.js

```diff
@@ -13,8 +13,8 @@ const codeValueTransaction = () => {
       facilityTransactionCodeValueCode: CONSTANTS.FACILITY.TRANSACTION_CODE.TYPEA,
     };
   } catch (error) {
-    console.error('Unable to map facility code value transaction record.', { error });
-    return error;
+    console.error('Unable to map facility code value transaction record. %O', error);
+    return {};
```

Comment:
Is there a reason for returning an empty object instead of the error? Same for below changes

Reply:
returning the error is a security risk so we need to either return a string error, or somewhere like this, not fully sure what to return

Reply:
Interesting - maybe return false boolean? Just my personal preference though, maybe empty object is better. Will have a think about error returns for EXIP 👍

Reply:
Agree `return false` is more acceptable then `return {}`.